packages/format/lib/bitcoin.ts

```diff
@@ -0,0 +1,37 @@
+import BigNumber from 'bignumber.js';
+
+/**
+ * Converts satoshis to BTC, rounds to the specified number of decimal places.
+ *
+ * @param sats
+ * @param decimalPlaces
+ * @returns
+ */
+export function toBTC(sats: number | string, decimalPlaces?: number): number {
+  const num = new BigNumber(sats).div(1e8);
```

Comment:
might be good to have a better error for when a non-number string is provided (double-check what error message from BigNumber is)